services/skvs-protonet.service

```diff
@@ -16,10 +16,10 @@ ExecStartPre=/usr/bin/docker run -d \
     --net=protonet \
     --volume=/etc/protonet:/data \
     -p 127.0.0.1:9200:82 \
-  quay.io/experimentalplatform/skvs:{{tag}} --exempt-from-cache system/channel
+  quay.io/experimentalplatform/skvs:development --exempt-from-cache system/channel
```

Comment:
Adjusted the entrypoint in SKVS image at https://github.com/experimental-platform/platform-skvs/commit/a47d7826402a36ec2a21e807f9d0f3987f9a9fe5